package.json

```diff
@@ -16,6 +16,8 @@
   "author": "",
   "license": "GPL-3.0",
   "dependencies": {
+    "@civicactions/data-catalog-services": "1.5.0",
```

Comment:
These are being removed as they are more trouble to maintain than value they add to the CMS project. In your downstream site, just comment out the body part of each template file until I am able to finish the rebuild of the data table in this repo with the new tanstack react table. If there is a missing service, we should just copy it over, I thought I got all that was needed moved in but I could have missed something.